apps/src/netsim/NetSimConnection.js

```diff
@@ -145,7 +153,9 @@ NetSimConnection.prototype.connectToShard = function (shardID, displayName) {
   }
 
   this.shard_ = new NetSimShard(shardID);
-  this.shardCleaner_ = new NetSimShardCleaner(this.shard_);
+  if (this.enableCleanup_) {
+    this.shardCleaner_ = new NetSimShardCleaner(this.shard_);
+  }
```

Comment:
Basically, I added a URL param that lets a client opt out of the cleanup routine, which can mess with the UI tests.